core/src/test/java/org/apache/commons/vfs2/provider/ram/test/CustomRamProviderTest.java

```diff
@@ -101,7 +101,7 @@ public void testSmallFS() throws Exception
             os.close();
             fail("It shouldn't save such a big file");
         }
-        catch (FileSystemException e)
+        catch (Exception e)
```

Comment:
Hi Manu. Thanks for pointing out this. I added this to avoid the test failures on the java8 due to java.lang.IllegalArgumentException: Self-suppression not permitted error. This is not a fixed for this issue and we are working on actual fix for this.